x/dispensation/types/keys.go

```diff
@@ -23,10 +23,11 @@ const (
 	QuerierRoute       = ModuleName
 	DefaultParamspace  = ModuleName
 	MaxRecordsPerBlock = 10
-	TokenSupported     = "rowan"
 )
 
 var (
+	// TODO : Refactor to genesis state
```

Comment:
@Jedi2002 should we address this TODO here?

Reply:
Good point , We can address this as a separate PR though. 
@marshsif  can you comment wether this feature , to support coins other than rowan is required or not .